docs/GettingStarted.md

```diff
@@ -1 +1,27 @@
-# Getting Started
\ No newline at end of file
+# Getting Started
+
+Thank you for using the component library. This page provides the information you'll need to get up and running.
```

Comment:
"The library's components are compliant with Government Design System (GDS) and meet the [WCAG 2.2](https://www.w3.org/TR/WCAG22/) Accessibility Guidelines." Since this is the first place people are directed to, we should have a comment like this (this is the from the README) which highlights the benefits of the library straight away